.github/workflows/generate-resources-file.yaml

```diff
@@ -1,11 +1,13 @@
 name: Generate Monarch Resources File
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
why? We only want the resource files to be generated when there is a push to master I think